### ISSUE TITLE
`FileInput` does not support keyboard operation

### ISSUE THREAD

**Issue by hirotomoyamada** [yamada-ui/yamada-ui#3126]
### Which package has an issue?

@yamada-ui/react

### Package Version

1.5.4

### Description

FileInput` does not support keyboard operation.

For example, it should be possible to select a file with enter or space while it is in focus.

### Steps to Reproduce

2. Click on '...'
3. Scroll down to '...'

### Browser

None

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux
- [ ] iPadOS
- [ ] iOS
- [ ] Android

### Do you know how to fix the issue?

Yes

### Are you willing to participate in fixing this issue and create a pull request with the fix?

Yes
